tests/integration/test_stepfunctions.py

```diff
@@ -18,6 +17,8 @@
 TEST_LAMBDA_NAME_1 = "lambda_sfn_1"
 TEST_LAMBDA_NAME_2 = "lambda_sfn_2"
 TEST_RESULT_VALUE = "testresult1"
+TEST_RESULT_VALUE_2 = "testresult2"
+TEST_RESULT_VALUE_4 = "testresult4"
```

Comment:
Guess I'll have to refactor this file at some point anyway.  Then we can get rid of these :sweat_smile:

Reply:
Yes :sweat_smile: I just added the new ones to distinguish between the different lambdas being executed.